test/system/profiles_test.rb

```diff
@@ -0,0 +1,83 @@
+require 'application_system_test_case'
+
+class ProfilesTest < ApplicationSystemTestCase
+  def setup
+    @superadmin = create :user_with_profile
+    @noprofile = create :user
+  end
+
+  test 'when first login displays profile form' do
+    visit new_user_session_path
+    fill_in 'Email', with: @noprofile.email
+    fill_in 'Password', with: 'asdfasdf'
+    click_on 'Log in'
```

Comment:
When you know what it is use the more specific method to click, in this case `click_link`but good to know, that there is something to click whatever, cool